chain/vectors/vectors_test.go

```diff
@@ -26,7 +26,6 @@ func LoadVector(t *testing.T, f string, out interface{}) {
 }
 
 func TestBlockHeaderVectors(t *testing.T) {
-	t.Skip("we need to regenerate for beacon")
```

Comment:
Have we done the vector regeneration described here?  
Do we even run vector tests any more?

Reply:
The tests pass.